### PR TITLE
Fixed issue #6394

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#6394 

## Description
Added a scroller on the empty input field on Add to cart button click

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Not working properly. the error should be focused only after one click.

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul  FIxed

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Is it **possible** to fix this for the **complete** velocity package?

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Please resolve the **conflicts**

File: packages/Webkul/Velocity/src/Resources/views/shop/products/view.blade.php [hunk 1]
```diff
@@ -286,9 +297,25 @@
                             setTimeout(function() {
                                 document.getElementById('product-form').submit();
                             }, 0);
+                        } else {
+                            this.activateAutoScroll()
                         }
                     });
                 },
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/products/view.blade.php:304]: add new line here

**Comment by akathuria289** [comment]
Added

**Pull request closed without merging**